cleanlab/multilabel_classification/filter.py

```diff
@@ -0,0 +1,243 @@
+import warnings
+from typing import Optional, Union, Tuple, List, Any
+import numpy as np
+
+
```

Comment:
Please make sure you also update the tutorial:
https://docs.cleanlab.ai/stable/tutorials/multilabel_classification.html

to use this new function

Reply:
I've updated the notebook
TODO: Once this PR is merged, I'll update the examples notebook for CelebA in a follow-up PR with the required changes